src/components/navigation/navigation.tsx

```diff
@@ -23,14 +23,24 @@ export class Navigation {
   /** Item links on the right side of the navigation. On vertical orientation, it will be added in order after primary-items. */
   @Prop() secondaryItems: any;
 
+  @Prop() caption: string;
```

Comment:
looks great!
Can you add a little documentation here for these props? So it will appear in the storybook properties description